<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
    </node>
    <node name="detect_vest" pkg="robot_tracker" type="detect_vest.py"
          args="--debug true"/>
<!--    <node name="vest_move_robot" pkg="robot_tracker" type="vest_move_robot.py" output="screen"/>-->
<!--    <node name="turtlesim" pkg="turtlesim" type="turtlesim_node"/>-->
</launch>
